packages/react-components/react-toast/src/state/vanilla/dispatchToast.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
```

Comment:
```suggestion
```

It's inside "state/**vanilla**/dispatchToast.ts", I don't think that React should leak there